src/exampleTypes/TermsStatsTable.js

```diff
@@ -5,14 +5,40 @@ import { observer } from 'mobx-react'
 import { exampleTypes } from 'contexture-client'
 import injectTreeNode from '../utils/injectTreeNode'
 import ExpandableTable, { Column } from '../layout/ExpandableTable'
+import { Flex } from '../layout/Flex'
+import Select from '../layout/Select'
 
+let toolBarStyle = { justifyContent: 'space-between', alignItems: 'center' }
+let SimpleLabel = ({ text }) => (
+  <label style={{ paddingRight: '5px' }}>{text}</label>
+)
 let SimpleFilter = observer(({ Input = 'input', ...props }) => (
-  <div>
-    Filter:
+  <Flex style={{ ...toolBarStyle, width: '75%' }}>
+    <SimpleLabel text="Filter:" />
     <Input type="text" {...props} />
-  </div>
+  </Flex>
+))
+let SelectSize = observer(({ node, tree }) => (
+  <Flex style={toolBarStyle}>
+    <SimpleLabel text="Size:" />
+    <Select
+      onChange={e => {
+        tree.mutate(node.path, { size: e.target.value })
+      }}
+      value={_.getOr(25, 'size', node)}
+      placeholder={null}
+      style={{ width: '100px' }}
+      options={_.map(x => ({ value: x, label: x }), [
```

Comment:
I think it makes sense this to be a parameter/prop with a these default values that TermsStatsTable can pass.

Reply:
Done